bin/circleci/run-integration-tests.sh

```diff
@@ -12,7 +12,7 @@ done
 python integration/runtests.py \
     --binary=$HOME/firefox/firefox-bin \
     --verbose \
-    integration
+    integration/test_installation.py
```

Comment:
Or rather, I'm hoping to figure out a working glob pattern while writing the second test